extensions/oauth/src/test/resources/operations/operations-config.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:oauth="http://www.mulesoft.org/schema/mule/oauth"
+      xsi:schemaLocation="
+       http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+       http://www.mulesoft.org/schema/mule/oauth http://www.mulesoft.org/schema/mule/oauth/current/mule-oauth.xsd">
+
+    <flow name="accessTokenFlow">
```

Comment:
not sure what `extensions client` means, but I replicated what I saw here https://github.com/mulesoft/mule/blob/mule-4.x/extensions/oauth/src/test/java/org/mule/test/oauth2/internal/tokenmanager/InvalidateOauthContextTestCase.java#L39 (seems more like a functional test, as you are using the complete model: xml -> parsers -> extension -> operation).